### ISSUE TITLE
[taskbar] Taskbar did not take in account background image position

### ISSUE THREAD

**Issue by GoogleCodeExporter** [roundcubevnz/myroundcube#299]
```
> What steps will reproduce the problem?
1. install taskbar plugin
2. try to create or edit an entry
3. change the 'position' from '5px 5px' to something else in config.inc.php

> What is the expected output? What do you see instead?

The background image (logo) must move based on the new position

> What version of the product are you using?
Roundcube-0.7.1 - Bundle v1.6

This is probably due because tasckbar.css.php did not read the 'position' field 
and have a background-position hardcoded to '5px 5px'

>How to correct
in tasckbar.css.php replace

  background-position: 5px 5px;

by

  background-position: " . $rcmail_config['login_taskbar_def'][$key]['position'] . ";
```

Original issue reported on code.google.com by `miniw...@gmail.com` on 21 Feb 2012 at 9:30

**Comment by GoogleCodeExporter** [comment]
Original comment by `rol...@liebl.ath.cx` on 25 Feb 2012 at 7:11
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
Original comment by `rol...@liebl.ath.cx` on 27 Feb 2012 at 8:14
- Changed title: **[taskbar] Taskbar did not take in account background image position**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in bundle v1.7.

Note: Plugin requires re-configuration.
```

Original comment by `rol...@liebl.ath.cx` on 1 Mar 2012 at 4:59

**Comment by GoogleCodeExporter** [comment]
Original comment by `rol...@liebl.ath.cx` on 1 Mar 2012 at 4:59
- Changed state: **Fixed**
